lib/PuppeteerSharp/PuppeteerSharp.csproj

```diff
@@ -29,13 +29,12 @@
   </PropertyGroup>
   <Import Project="../Common/CommonProps.props" />
   <ItemGroup>
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.NetAnalyzers" Version="8.0.0">
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.2" />
```

Comment:
Do we need this library for .net8.0?

Reply:
Do you mean upgrading it to v8?

Reply:
I wasn't on my dev machine, so I was just unsure if the types in this package are already available in net8